### PR TITLE
[14.0] product variant attributes to template

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

File: product_import_autocomplete_attribute/README.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+Configuration
+=============
+
+Create some product attributes
+
+Usage
+=====
+
+In order to better test and illustrate this module, and example CSV file is provided in the exmaple folder (you will need the demo data).
```
Comment (rvalyi) [inline product_import_autocomplete_attribute/README.rst]:
```suggestion
In order to better test and illustrate this module, and example CSV file is provided in the example folder (you will need the demo data).
```

File: product_import_autocomplete_attribute/tests/test_fill_attribute_line.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# © 2020 TestVariantAttrToTmplAkretion France
```
Comment (rvalyi) [inline product_import_autocomplete_attribute/tests/test_fill_attribute_line.py]:
```suggestion
# © 2020 Akretion France
```

**Comment by sebastienbeau** [comment]
@rvalyi can you update your review ?

File: product_import_autocomplete_attribute/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
+In order to better test and illustrate this module, and example CSV file is provided in the example folder (you will need the demo data).
```
Comment (rvalyi) [inline product_import_autocomplete_attribute/readme/USAGE.rst:1]: s/and example CSV /an example CSV

File: product_import_autocomplete_attribute/models/product_template.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    def _get_existing_variant(self, values):
+        self.ensure_one()
+        for record in self.with_context(active_test=False).product_variant_ids:
+            if record.attribute_value_ids == values:
+                return record
+        else:
+            raise UserError(
+                _(
+                    "No matching variant found, maybe the attribute value "
```
Comment (rvalyi) [inline product_import_autocomplete_attribute/models/product_template.py:47]: s/attribute value are incoherant/attribute values are inconsistent/

**Pull request closed without merging**